### DISCUSSION TITLE
Ye, idk how use godot .. help

### DISCUSSION THREAD

**Discussion by Sobular** [Techokami/SonicWorldsNext#186] [Q&A] [answered]
ayo! somebody can help me with this? i just download the engine for Godot 3, when i try to run the scene, or the game itself. This error pop off .. help me pls (the error said basically, the location of the file that is highlighted in the code is not found. Which I don't know why, since I never touched anything)

![Screenshot 2024-08-29 18 40 28](https://github.com/user-attachments/assets/689e5e53-3abc-44d6-ba82-b9072743d558)

**Comment by Caverns4** [comment]
I would strongly advise using the Godot 4.2 version as the 3 version is unmaintained and very outdated.
That said, if you want to use the Godot 3 version, the only advice I can offer as I wasn't active at that time is to close Godot and try deleting the .godot folder (the cache), then reload the project. Hope this helps.

**Reply by Sobular** [reply]
morning' , Well, since I couldn't find the .godot folder, I decided to use Godot 4.3 instead, although I don't know if that causes errors, but so far I haven't had any... and is there a tutorial on how to use the engine?

**Comment by Caverns4** [comment, marked as answer]
Godot 4.3 isn't fully tested with Sonic Worlds Next, but I've never had any issues with it.
That said, if this is your very, very first time using Godot, this isn't really the best place to go. I'd recommend pulling up a Tutorial for 4.2 to learn the basics. Youtbe channel BornCG has an ongoing series for the basics Godot 4.2, which is nearly identical to 4.3 other than just some polishing tweaks. It's how I learned to use it.

**Comment by Sobular** [comment]
evenin'. Gotcha, thanks man for you help!!
